tests/scenarios/cp/eval.py

```
@@ -0,0 +1,262 @@
+import json
+import os
+import subprocess
+import sys
+import warnings
+
+warnings.filterwarnings("ignore", category=SyntaxWarning)
+
+from common.logging import get_logger
```

Ruff (0.11.9)

Module level import not at top of file

(E402)